crates/red_knot_python_semantic/src/types.rs

```diff
@@ -147,6 +147,12 @@ enum AttributeKind {
     NormalOrNonDataDescriptor,
 }
 
+impl AttributeKind {
```

Comment:
This is a drive-by change, just moving the `impl` for `AttributeKind` next to the struct definition.